React/Modules/RCTAnimatedNode.m

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import "RCTAnimatedNode.h"
+
+@implementation RCTAnimatedNode {
+  NSMutableDictionary *_childNodes;
```

Comment:
Please use lightweight generics for all collection types, it makes the code much easier to understand, e.g.

    NSMutableDictionary<NSNumber *, RCTAnimatedNode *> *_childNodes;